code/modules/admin/admin_investigate.dm

```diff
@@ -49,50 +49,9 @@
 				to_chat(src, "<font color='red'>Error: admin_investigate: Href Logging is not on.</font>")
 				return
 
-		if("pda")			//general one-round-only stuff
-			var/F = investigate_subject2file(subject)
-			if(!F)
-				to_chat(src, "<font color='red'>Error: admin_investigate: [INVESTIGATE_DIR][subject] is an invalid path or cannot be accessed.</font>")
-				return
-			src << browse(F,"window=investigate[subject];size=800x300")
-
-		if("cult")
-			var/F = investigate_subject2file(subject)
-			if(!F)
-				to_chat(src, "<font color='red'>Error: admin_investigate: [INVESTIGATE_DIR][subject] is an invalid path or cannot be accessed.</font>")
-				return
-			src << browse(F,"window=investigate[subject];size=800x300")
-
-		if("atmos")
-			var/F = investigate_subject2file(subject)
-			if(!F)
-				to_chat(src, "<font color='red'>Error: admin_investigate: [INVESTIGATE_DIR][subject] is an invalid path or cannot be accessed.</font>")
-				return
-			src << browse(F,"window=investigate[subject];size=800x300")
-
-		if("ntsl")
```

Comment:
To be clear, its presence here doesn't necessarily mean anything was being logged with the "ntsl" subject, just that it was a subject at one point in time. Maybe.